cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricTestHelper.java

```diff
@@ -155,12 +155,12 @@ public static void ensureNamespaceExists(Id.Namespace namespace) throws Exceptio
   private static void ensureNamespaceExists(Id.Namespace namespace, CConfiguration cConf) throws Exception {
```

Comment:
can you check if this method can accept ``NamespaceId`` without too many changes?